test/gtest/common/test_helpers.h

```diff
@@ -390,6 +390,10 @@ static inline int rand() {
     return ::rand();
 }
 
+static inline int rand_range(int max) {
+    return rand() % max;
+}
```

Comment:
```suggestion
static inline int rand(int min = std::numeric_limits<int>::min(), int max = std::numeric_limits<int>::max()) {
    return min + (rand() % (max - min + 1));
}
```

Reply:
@dmitrygx i think it's not needed for now since min since range is 0..max-1
(also std::numeric_limits<int>::min() will start from some big negative number and we want to start from 0)